platform/ios/CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 ### Annotations
 
+* MGLPolyline and MGLPolygon annotations are now able to be mutated, part or all, and changes are displayed immediately. ([#6565](https://github.com/mapbox/mapbox-gl-native/pull/6565))
```

Comment:
Since we don’t respond to mutations to inner polygons, let’s say that MGLPolyline and the exterior of MGLPolygon are mutable.